app/views/clients/_request.html.slim

```diff
@@ -0,0 +1,10 @@
+tr
+  td= t('volunteer_request')
+  td
+    - if client.gender_request.present? && client.age_request.present?
+      dl.dl-horizontal
+        dt= t_attr(:gender_request)
+        dd= client.gender_request
+
+        dt= t_attr(:age_request)
+        dd= client.age_request
```

Comment:
no newline at end of file. Install editorconfig plugin to your editor, and you'll never even notice it beeing done right.... ;)